services/web-app/config/constants.js

```diff
@@ -4,5 +4,5 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-export const CACHE_PATH = '.carbon'
+export const CACHE_PATH = process.env.FILE_SYSTEM_CACHE || '.carbon-oss'
```

Comment:
Since we don't expect these to change, I'd recommend:

```js
import { getRunMode, DEV, PROD } from '@carbon-platform/api/run-mode'

export const CACHE_PATH = getRunMode() === PROD ? '.carbon' : '.carbon-oss'
```